src/components/DocExplorer.js

```diff
@@ -496,7 +496,7 @@ class TypeDoc extends React.Component {
                 {': '}
                 <TypeLink type={field.type} onClick={onClickType} />
                 {
-                  field.isDeprecated &&
+                  field.deprecationReason &&
```

Comment:
@leebyron Hmm. Yeah, I really don't know -- definitely not removed. 

Digging through commits here and in `graphql-js` I don't see anything that would have caused this to change recently. I may be that this never worked as advertised.

In any case, I think checking the presence of either of these fields is what we want since `deprecationReason` is an optional field.

I've resolved this in https://github.com/graphql/graphiql/pull/165/commits/0aa91637d26bb98a217eed17cb6f115f28175239